integration-tests/src/tests/nearcore/run_nodes.rs

```diff
@@ -1,9 +1,9 @@
-use actix::{Actor, System};
-use futures::{future, FutureExt};
+use std::ops::ControlFlow;
+
+use actix::System;
 
```

Comment:
As we are touching imports here anyway, let's remove blank lines, as per our styleguide: 

https://github.com/near/nearcore/blob/master/docs/style.md#import-blocks